<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_one">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>651</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_one">
     <attribute name="title">
      <string>Extract Pages</string>
     </attribute>
     <widget class="QListWidget" name="fileList_two">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>256</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="uploadButton_two">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Load PDF</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pagesList">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>1,2,6-8</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Pages</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="outputFilename">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output file name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="extractButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="outputPathButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Merge PDFs</string>
     </attribute>
     <widget class="QPushButton" name="removeButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove All</string>
      </property>
     </widget>
     <widget class="QListWidget" name="fileList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>256</width>
        <height>131</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="uploadButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add PDF(s)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mergeButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Merge Files</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="outputFilename_two">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output file name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="outputPathButton_two">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
